### PR TITLE
RD-5575 Fix user_spec

### PR DESCRIPTION
**Pull request by leon-good-life**
## Description
Jira ticket: https://cloudifysource.atlassian.net/browse/RD-5575

## Screenshots / Videos
Screen Shot 2022-08-09 at 10 59 40

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests
https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/3412/pipeline

 

## Documentation
N/A

### CODE REVIEW

**Comment by leon-good-life** [comment]
The test passed on Jenkins.

File: test/cypress/integration/flows/user_spec.ts [hunk 1]
```diff
@@ -53,6 +53,7 @@ describe('User flow', () => {
         });
         cy.get('.modal', { timeout: minutesToMs(1) }).should('not.exist');
 
+        cy.visitPage('Blueprints');
         cy.getSearchInput().clear().type(resourceName);
         cy.clickButton('Deploy');
         cy.get('input[name=deploymentName]').type(resourceName);
```
Comment (qooban) [inline test/cypress/integration/flows/user_spec.ts:59]: This test is for checking user flow with default templates. 

Right now the flow allows you to enter Create deployment modal just after uploading it without any additional actions, so why do we want to visit Blueptrints page if we are already in Create Deployment modal? I would consider removing L56-L58.

Comment (leon-good-life) [reply test/cypress/integration/flows/user_spec.ts]: [2db5869](https://github.com/cloudify-cosmo/cloudify-stage/pull/2125/commits/2db58691f4a7f9c91e066f37fc8e0d188bf79916)

**Comment by leon-good-life** [comment]
I verified locally:
Screen Shot 2022-08-09 at 11 31 35

I will run on Jenkins too.

**Comment by leon-good-life** [comment]
Jenkins: https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/3413/pipeline

**Comment by leon-good-life** [comment]
The test passed

**Review by qooban — APPROVED** [review]

**Pull request merged**